### PR TITLE
=doc Fix #1129: Use paradox akka-theme artifacts from jcenter

### LINKED ISSUE #1129: Update documentation to use the new Akka theme and paradox plugin
See related bits from the Akka repo:

 - https://github.com/akka/akka/blob/wip-akka-docs-new/project/plugins.sbt
 - https://github.com/akka/akka/blob/wip-akka-docs-new/akka-docs/build.sbt

### PR DESCRIPTION
**Pull request by jonas**
Based on akka/akka#22987.

---

Fixes #1129 by upgrading to first officially released version.

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -4,11 +4,10 @@ resolvers += Resolver.sonatypeRepo("releases") // to more quickly obtain paradox
 
 // need this to resolve http://jcenter.bintray.com/org/jenkins-ci/jenkins/1.26/
 // which is used by plugin "org.kohsuke" % "github-api" % "1.68"
-resolvers += "Bintray Jcenter" at "https://jcenter.bintray.com/"
+resolvers += Resolver.jcenterRepo
 
 // for sbt-bintray for resolving credentials from env vars
-resolvers += Resolver.url("2m-sbt-plugin-releases", url("https://dl.bintray.com/2m/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
-resolvers += Resolver.bintrayRepo("2m", "sbt-plugin-releases")
+resolvers += Resolver.bintrayIvyRepo("2m", "sbt-plugin-releases")
```
Comment (raboof) [inline project/plugins.sbt]: Is this still needed? sbt-plugin-releases should be in https://bintray.com/sbt/sbt-plugin-releases which I *thought*  comes by default with sbt (but haven't checked)

Comment (jonas) [inline project/plugins.sbt]: Yes, because Akka HTTP uses a modified version of sbt-bintray pending https://github.com/sbt/sbt-bintray/pull/69

Comment (jonas) [inline project/plugins.sbt]: Also this is Martynas Mickevičius' sbt-plugin-releases repo.

Comment (raboof) [inline project/plugins.sbt]: ahh yes you're right I ran into the same thing for akka-paradox - hopefully this will be a thing of the past after sbt 1.0 :)

**Pull request merged**